app/controllers/api/v1/plants_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 class Api::V1::PlantsController < ApiController
   before_action :set_plant, except: %i[index create]
+  skip_before_action :doorkeeper_authorize!, only: :index
+  skip_after_action :verify_authorized, only: :index
```

Comment:
pas besoin de skip ici parce que tu appelles quand même `authorize plants` dans ton index.

Même si la méthode se contente de renvoyer `true` dans la policy, ce n'est pas plus mal de systématiquement garder les autorisation "au cas où", IMO. Ça laisse le code plus constant en forçant à utiliser les `authorize` partout, et ça permet d'éviter d'oublier de le remettre plus tard si jamais la spec doit changer ! :) 

En revanche, c'était effectivement la bonne chose à faire que de skip le `doorkeeper_authorize!` pour rendre la route publique 👍